strr-examiner-web/app/pages/dashboard.vue

```diff
@@ -135,7 +135,7 @@ const { data: applicationListResp, status } = await useAsyncData(
         requirements: getRequirementsColumn(app),
         applicantName: getApplicantNameColumn(app),
         propertyAddress: getPropertyAddressColumn(app),
-        status: app.header.examinerStatus || app.header.hostStatus, // TODO: should this have registration status? maybe this should just return app.header.status?
+        status: app.header.registrationStatus ? app.header.examinerStatus : app.header.hostStatus, // TODO: should this have registration status? maybe this should just return app.header.status?
```

Comment:
Shouldn't this always be examinerStatus?

Reply:
From the designs it looks like for the Approved applications the examiner status showing, but for others, host status. I will ask Andy to verify once we'll deploy to Dev and be able to see all statuses in the table.